includes/entity_plus.controller.inc

```diff
@@ -693,7 +693,6 @@ class EntityPlusController extends DefaultEntityController implements EntityPlus
       '#entity_plus_wrapped' => $wrapper,
       '#theme' => 'entity_plus_property',
       '#property_name' => $name,
-      '#access' => $property->access('view'),
```

Comment:
@argiepiano I was fixing merge conflicts and happened to see this. No problem removing this one, right? (It's superceded by another `'#access' property that you added further down in the array, I think).

Reply:
@laryn I didn't add this line - it was there before. In theory the correct line is the one that invokes the method `->access('view')`, since you want to check for access when viewing. The duplicate line simply returns TRUE if the property has any value, without checking for access. Someone must have put that line there for a reason. For the time being I'd say to keep the second #access line and remove the one you removed, but at some point we'll need to check why that was done and potentially revert to checking access.

Reply:
I'm having second thoughts about removing this. While with the current code the first '#access' is superseded by the second one, I believe it's important to "merge" both lines and still invoke the access method from the wrapper. This access method will invoke the function defined in access callback for the property, if defined, or return TRUE for view if it's not defined.